Add Code to Cloud U9 release notes (#8861)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

### Improvements

#### `cloud` package

> - Directories can now be mounted as Configmaps and Secrets 

Reviewer: ```suggestion
- Directories can now be mounted as Configmaps and Secrets.
```

### Bug fixes

## Backward-incompatible changes

### `cloud` package

#### Cloud.toml related changes

> - SSL configurations are not automatically retrieved from the code anymore. You need to explicitly mark them as secrets in Cloud.toml. 

Reviewer: ```suggestion
- SSL configurations are no longer automatically retrieved from the code. You need to explicitly mark them as secrets in `Cloud.toml`. 
```

> - SSL configurations are not automatically retrieved from the code anymore. You need to explicitly mark them as secrets in Cloud.toml. 

Reviewer 2: We usually just have bullet points here, but then again it's usually just lang.

How about we add `### `cloud` package` (and similarly for language changes, we will add a heading) and then just have bullet points without another level of headers?

Reviewer 2: I've suggested this in https://github.com/ballerina-platform/ballerina-dev-website/pull/8876/files#diff-7ee6bb79765f14c3dff835d98a4c33179eedea8962b3fc2da71bb5670d0ff67dR125

```toml
[[cloud.secret.files]]
file="resource."
mount_dir="./resource"
```

##### [[cloud.secret.files]] and [[cloud.config.maps]] changes

> - `mount_path` is renamed to `mount_dir` fields of in Cloud.toml file and now it always expects the destination directory.

Reviewer: ```suggestion
- The `mount_path` is renamed to `mount_dir` fields in the `Cloud. tool` file, and now it always expects the destination directory.
```

> - `mount_path` is renamed to `mount_dir` fields of in Cloud.toml file and now it always expects the destination directory.

Reviewer 2: ```suggestion
- `mount_path` is renamed as `mount_dir` in the `Cloud.toml` file and now it always expects the destination directory.
```

##### Potential breaking changes for Kustomize users

> - Entrypoints are used instead of CMD to run the ballerina application in the dockerfile.

Reviewer: ```suggestion
- Entrypoints are used instead of CMD to run the ballerina application in the Dockerfile.
```

```
CMD ["java","..."] //Old
```

```
ENTRYPOINT ["java","..."] //New
```

> - Suffix is added to generated Config Maps and Secrets in Kubernetes to avoid Conflicts.

Reviewer 2: Config Maps vs ConfigMaps?

> - Subpaths are used in kubernetes to better support multiple files in the same directory.

Reviewer: ```suggestion
- Subpaths are used in Kubernetes to better support multiple files in the same directory.
```

### Improvements

> #### `cloud` package

Reviewer 2: Is this under the correct section?
